.github/workflows/container-build-push-2nd-gen.yml

```diff
@@ -121,9 +129,13 @@ jobs:
             type=semver,pattern={{major}},enable=${{ github.event_name != 'workflow_dispatch' }}
             type=edge
             type=ref,event=pr
-            type=raw,value=latest,enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}
-            type=raw,value=stable,enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}
-            type=raw,value=edge,enable=${{ (inputs.ref-name || github.ref_name) == 'main' }}
+            type=raw,value=latest,enable=${{ (steps.latest.outputs.is-latest-tag == 'true') && (inputs.base-image-label == 'stable') }}
+            type=raw,value=stable,enable=${{ (steps.latest.outputs.is-latest-tag == 'true') && (inputs.base-image-label == 'stable') }}
+            type=raw,value=edge,enable=${{ ((inputs.ref-name || github.ref_name) == 'main') && (inputs.base-image-label == 'stable') }}
+            type=raw,value=oldstable,enable=${{ (steps.latest.outputs.is-latest-tag == 'true') && (inputs.base-image-label == 'oldstable') }}
+            type=raw,value=oldstable-edge,enable=${{ ((inputs.ref-name || github.ref_name) == 'main') && (inputs.base-image-label == 'oldstable') }}
+            type=raw,value=testing,enable=${{ (steps.latest.outputs.is-latest-tag == 'true') && (inputs.base-image-label == 'testing') }}
+            type=raw,value=testing-edge,enable=${{ ((inputs.ref-name || github.ref_name) == 'main') && (inputs.base-image-label == 'testing') }}
```

Comment:
```suggestion
            type=raw,value=temp # temporary tag that will be overwritten with the manifest upload
```

Reply:
@pascalholthaus suggested:
` image-tags: type=raw,value=${{ inputs.ref_name }} # temporary tag that will be overwritten with the manifest upload`

Reply:
I would use something like ``` raw,value=${{ inputs.ref-name || github.ref_name }}-amd64/arm64```